manage

```diff
@@ -682,6 +683,18 @@ test.pylint() {
     dump_return $?
 }
 
+
+test.pyright() {
+    build_msg TEST "[pyright] static type check of python sources"
+    nodejs.ensure
+    [ ! -d node_modules ] && npm install
+    # We run Pyright in the virtual environment because Pyright
+    # executes "python" to determine the Python version.
+    pyenv.cmd npx --no-install pyright -p pyrightconfig-ci.json
```

Comment:
The `pyrightconfig.json` is there because it's automatically detected by the Pyright extension for VS Code (and I assume also Pyright extensions for other editors). When viewing the files in your editor you want all errors to be underlined with red squiggly lines.

Reply:
> because when viewing the files in your editor you want all errors to be reported 

Ah, OK .. make sense!